doc/guide/admin_slurm.rst

```diff
@@ -15,41 +15,69 @@ Check back with us at the end of 2026.
+as a system workload manager until 2024 with the early deliveries of
+`El Capitan <https://hpc.llnl.gov/hardware/compute-platforms/el-capitan>`_.
+
+Flux is now in daily production use as the sole system workload manager on
+El Capitan (currently in slot 1 of the `TOP500 <https://www.top500.org/>`_)
+and its unclassified sister systems at LLNL.  These machines are capability
+workhorses, continuously in demand for LLNL's most cutting edge,
+mission-critical activities.  The process of standing them up brought the
+Flux team together to address problems of scale and stability with
+unprecedented urgency.  As a result, system deployments of Flux on
+virtually any size system are viable at this point.
+
+This experience did bring missing features in to focus that have now been
+prioritized for near term development.  We expect substantial progress to
+be made on all of these items in 2026:
+
+Rolling software upgrade
+  Flux nodes won't interact with other Flux nodes unless they are running
+  *exactly* the same flux-core version.  When flux-core 1.0.0 is released,
+  relaxed rules will be enacted to support rolling upgrades.
+
+Flux restart with running jobs
+  Restarting Flux kills running jobs.  The design to allow running
+  jobs to continue is not yet fully implemented.
+
+Reservations
+  Flux does not yet have a way to request allocations that begin at a fixed
+  time in the future.  Arrangements for dedicated application time currently
+  require manual/scripted actions by system administrators.
+
+Flux graph-based scheduling scalability
+  Efforts to optimize the Fluxion scheduler graph-based resource
+  representation for storage space efficiency are ongoing.  Meanwhile a
+  flat resource model similar to Slurm's that scales well but is limited
+  in capability is typically used on large systems.
+
+Preservation of job step information
+  In Flux, there are no job steps.  Instead, a sub-instance of Flux is started
+  on a resource allocation, and user applications are run as jobs within the
+  sub-instance.  There is little coupling between the system instance and its
+  sub-instances which improves scalability.  Unfortunately, sub-instance job
+  data is not preserved unless the user explicitly arranges for it, which some
+  find surprising.
+
+Multi-system fair share accounting
+  Flux has an optional fair-share accounting system for gathering usage data
+  and setting job priorities.  Unlike Slurm, there is not yet a capability to
+  deploy one accounting system for multiple Flux systems.
+
+Package availability
+  Flux system administrators are encouraged to install Flux as system packages,
+  but pre-built packages are not widely available.  Source RPM packages for
+  RHEL 8 and 9 are manually attached to GitHub releases, and test debs can be
+  built from the source trees of several Flux Framework projects.  Automated
+  package production and Linux distribution relationships are under discussion.
```

Comment:
Under discussion makes it sound like it may _not_ happen. I think it's more something we aspire/want to do, we just need to figure out the right way to do it. Can we say something like planned? On the roadmap for the coming year? Two years? Underway? Likely in the future?